Runtime/Scripts/Export/GameObjectExport.cs

```diff
@@ -107,6 +111,50 @@ void ExportSkyboxMaterial()
                 skyboxMaterial, out int skyboxMaterialId, m_MaterialExport);
         }
 
+        void ExportUnityMaterials()
+        {
+            string exportPath = Path.Combine(
+                Path.GetDirectoryName(Application.dataPath),
+                "Temp",
+                "unityMaterials.zip");
+
+            List<Material> materials = m_Writer.GetUnityMaterials();
+            using (FileStream fileStream = File.Open(exportPath, FileMode.Create))
+            {
+                using (ZipArchive archive = new ZipArchive(fileStream, ZipArchiveMode.Create))
+                {
+                    foreach (Material material in materials)
+                    {
+                        AddAssetToArchive(archive, material);
+
+                        Shader shader = material.shader;
+                        AddAssetToArchive(archive, shader);
+
+                        string[] textureNames = material.GetTexturePropertyNames();
+                        foreach (string textureName in textureNames)
+                        {
+                            if (material.HasProperty(textureName))
+                            {
+                                UnityEngine.Texture texture = material.GetTexture(textureName);
+                                if (texture != null)
+                                    AddAssetToArchive(archive, texture);
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        void AddAssetToArchive(ZipArchive archive, UnityEngine.Object asset)
+        {
+            string path = AssetDatabase.GetAssetPath(asset);
+            if (path.Contains("Packages/com.unity.render-pipelines"))
```

Comment:
Maybe should add handling for the other default materials as well - not just the URP Lit shader.